pkgs/os-specific/darwin/libtapi/default.nix

```diff
@@ -10,7 +10,10 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ cmake python3 ];
-  buildInputs = [ clang_6.cc ];
+
+  # ncurses is required here to avoid a reference to bootstrap-tools, which is
+  # not allowed for the stdenv.
+  buildInputs = [ ncurses clang_6.cc ];
```

Comment:
We should really get this to build with the current compiler if it's enabled by default, bootstrapping multiple versions is a huge pain and llvm 6 is already quite old at this point.

Reply:
Looking closer, I don't think clang is intended to be an external dependency. I've pushed an update but I'm still waiting for it to test building stdenv locally.